runelite-client/src/main/java/net/runelite/client/plugins/runenergy/RunEnergyOverlay.java

```diff
@@ -90,6 +91,11 @@ public Dimension render(Graphics2D graphics)
 				sb.append("Run Time Remaining: ").append(plugin.getEstimatedRunTimeRemaining(false));
 			}
 
+			if (client.getVarbitValue(Varbits.RUN_SLOWED_DEPLETION_ACTIVE) == 0 && plugin.getRingOfEnduranceCharges() == null && plugin.isRingOfEnduranceEquipped())
+			{
+				sb.append("</br>Check your Ring of endurance to get the time remaining.");
```

Comment:
I don't think this makes sense to show `Run time remaining: xyz<br>Check your Ring of endurance to get the time remaining.` like you are doing here since it is self-contradictory. I would probably do what I had prior to your last change.

Reply:
The way it is now, the `xyz` value will always be a `?` when the message is showing.

This way we always inform the player why there's a `?` instead of a time. The way you had it before would only tell the user why there's a `?` if they had the replaceOrbText set to false.

Reply:
<img width="694" alt="image" src="https://user-images.githubusercontent.com/10326958/192048349-0a48444d-b895-44e5-b686-d0530cc4e1c5.png">

In both of these images there's a `?` and the message `Check your Ring of endurance to get the time remaining.`

What was there prior to my change would only show the message in one of these images, but the `?` both times. I hope this is making sense.